### PR TITLE
Add tag cleanup section

### PR DESCRIPTION
**Pull request by hanxiaop**
## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

**Comment by hanxiaop** [comment]
This is to sync parts in https://github.com/istio/istio.io/pull/14520

File: content/zh/docs/setup/upgrade/canary/index.md [hunk 1]
```diff
@@ -256,6 +256,13 @@ $ istioctl uninstall --revision=canary -y
 
 ## 清理 {#cleanup}
 
+1. 清理已创建的修订版本标签:
```
Comment (windsonsea) [inline content/zh/docs/setup/upgrade/canary/index.md:259]:
```suggestion
1. 清理已创建的修订版本标签：
```
可以改成中文标点

**Review by windsonsea — COMMENTED** [review]
one nit

File: content/zh/docs/setup/upgrade/canary/index.md (hunk 1 above)
Comment (hanxiaop) [reply content/zh/docs/setup/upgrade/canary/index.md]: Thanks

**Comment by hanxiaop** [comment]
> /test lint

Looks like the indention is wrong, I'll fix it.

File: content/zh/docs/setup/upgrade/canary/index.md [hunk 2]
```diff
@@ -256,6 +256,13 @@ $ istioctl uninstall --revision=canary -y
 
 ## 清理 {#cleanup}
 
+1. 清理已创建的修订版本标签：
+
+    {{< text bash >}}
```
Comment (windsonsea) [inline content/zh/docs/setup/upgrade/canary/index.md:261]: Yes, it's ok now. We normally use 4 spaces for indentation.

**Review by windsonsea — APPROVED** [review]
thanks, lgtm

**Pull request merged**